command/agent/agent.go

```diff
@@ -1361,3 +1361,15 @@ func (a *Agent) DisableNodeMaintenance() {
 	a.RemoveCheck(nodeMaintCheckID, true)
 	a.logger.Printf("[INFO] agent: Node left maintenance mode")
 }
+
+// Query sends a Query on Serf, see Serf.Query.
+func (a *Agent) Query(name string, payload []byte, params *serf.QueryParam) (*serf.QueryResponse, error) {
+	a.logger.Printf("[DEBUG] jfs: consul/command/agent/agent.go Query() invoked...")
```

Comment:
Let's clean up the jfses and replace them with something more meaningful (I think `agent` makes sense in this case).